src/test/java/core/basesyntax/DataParserServiceTest.java

```diff
@@ -0,0 +1,78 @@
+package core.basesyntax;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.DataParserService;
+import core.basesyntax.service.impl.DataParserServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class DataParserServiceTest {
+    private static List<FruitTransaction> excepted;
```

Comment:
```suggestion
    private List<FruitTransaction> expected;
```
same everywhere where needed